lib/pure/subexes.nim

```diff
@@ -384,15 +384,15 @@ when isMainModule:
 
   doAssert "$#($', '10c'\n    '{#..})" % ["doAssert", "longishA", "longish"] ==
            """doAssert(
-    longishA,
+    longishA, 
```

Comment:
That's fragile (we all hate trailing whitespace now), better use some smarter equality operation here.